server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -1175,4 +1174,20 @@ public int getConsumerEntitlementCount(Owner owner, ConsumerType type) {
             .list();
     }
 
+    /**
+     * Class for holding the list of consumers in the stored json text
+     *
+     * @author wpoteat
+     */
+    public static class HypervisorList {
```

Comment:
I moved it because it is part of the ConsumerCurator API. Method getHostConsumersMap expects HypervisorList as argument. It would be weird if you want to use getHostConsumersMap but to do so you need model from HypervisorUpdateJob.

Reply:
That curator method is only called by the HypervisorUpdateJob, and looking at it, the only thing it does with it is call `hypervisorList.getHypervisors()` which returns a list of Consumers. It would be better to simply move the HypervisorList back to the HypervisorUpdateJob and change `getHostConsumersMap` to receive a `List<Consumer>` it seems.

Reply:
Done.